src/SFA.DAS.RequestApprenticeTraining.Jobs/SFA.DAS.RequestApprenticeTraining.Jobs.csproj

```diff
@@ -25,6 +25,8 @@
     <PackageReference Include="NLog" Version="5.3.2" />
     <PackageReference Include="NLog.Extensions.Logging" Version="5.3.11" />
     <PackageReference Include="SFA.DAS.NLog.Targets.Redis" Version="1.2.1" />
+    <PackageReference Include="System.Drawing.Common" Version="8.0.8" />
```

Comment:
And these we are doing pretty much what we were doing before - odd that these are now required?

Reply:
Same as above